docs/content/concepts/automation/schedules.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: "Schedules | Dagster Docs"
+description: "Use schedules to run your pipelines at fixed time intervals."
+---
+
+# Schedules
+
+Schedules are Dagster's way of supporting traditional methods of [automation](/concepts/automation), which allow you to specify when a [job](/concepts/ops-jobs-graphs/jobs) should run. Using schedules, you can define a fixed time interval to run a pipeline, such as daily, hourly, or Monday at 9:00 AM.
+
+Each interval of a schedule is called a **tick**, which is an indication that a job should begin executing. Ticks kick off **runs**, which is a single instance of a job being executed.
```

Comment:
If this is the same "tick" as [freshness policy sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/asset-sensors#freshness-policy-sensors), I feel like my understanding is our "scheduler" has a tick, every <n> seconds, and in each tick it checks to see if the schedule should run, then if that tick occurs at the time indicated by the schedule, a run will be launched.

Does that align with this sentence?

Reply:
I'm inclined to say no - it's my understanding that whenever the current time aligns with a schedule's interval, a tick is created and then a run is launched.

The **Tick history** tab in the UI seems to support this: 
<img width="1217" alt="Screenshot 2024-05-31 at 10 45 10 AM" src="https://github.com/dagster-io/dagster/assets/16615212/d2942dea-7cf1-4bfe-b19a-ffeee0ba3844">

Reply:
Gotcha thank you for the clarification -- that makes sense to me.